cmd/checkmetrics/ci_slaves/checkmetrics-json-kata-metric2.toml

```diff
@@ -42,7 +42,7 @@ description = "measure container average footprint with KSM"
 # within (inclusive)
 checkvar = ".\"memory-footprint-ksm\".Results | .[] | .average.Result"
 checktype = "mean"
-midval = 55879.5
+midval = 55945.5
```

Comment:
My only request is you add some info to the commit about how/where you got the data and calculated the new value from (so others in the future can check, reproduce or follow the method).

See https://github.com/kata-containers/tests/pull/1376#issue-265813009 for an example.

It's quite hard to add too much information to our Issues, PRs and commits - but easy to not add enough ;-)

Reply:
@grahamwhaley changes applied